src/basic/time-util.h

```diff
@@ -1,6 +1,7 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 #pragma once
 
+#include <sys/timex.h>
```

Comment:
Please drop this. It's musl problem to fix, or the shims' problem to deal with.

Reply:
Same, time.h anyway includes sys/timex.h. Hence, this does not change anything.
This does not impact build performance.

Reply:
Fixed.